src/makeInfinityAdder.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const add = (args) => {
+    let resultSum = 0;
+
+    if (typeof (args) !== 'number') {
```

Comment:
suggestion: I'd rather do it this way
```suggestion
    if (typeof args === 'undefined') {
```